packages/components-react/src/paragraph/test/paragraph.spec.tsx

```diff
@@ -0,0 +1,34 @@
+import { describe, expect, it } from '@jest/globals';
+import { render, screen, within } from '@testing-library/react';
+import { LuxParagraph } from '../Paragraph';
+
+const paragraphText = 'Dit is een paragraaf';
+const leadParagraphText = paragraphText + ' die een inleiding is';
+const smallParagraphText = paragraphText + ' die klein is';
+
+describe('Paragraph', () => {
+  it('renders a paragraph', () => {
+    render(<LuxParagraph>{paragraphText}</LuxParagraph>);
+
+    const paragraph = screen.getByRole('paragraph');
+
+    const utrechtParagraph = within(paragraph).getByText(paragraphText, { selector: '.utrecht-paragraph' });
```

Comment:
Ik ging er eigenlijk van uit dat je wel zou begrijpen dat dit over het hele bestand van toepassing zou zijn. Dus om het expliciet te maken: kun je deze wijziging ook doorvoeren in de andere tests?

Reply:
Ik had hem via Github toegepast zodat ik hem daarna op de andere kon overnemen...